load.go

```diff
@@ -564,7 +564,7 @@ func validatePatch(patch PatchSpec, patchSrc Source) error {
 }
 
 func (g *SourceGenerator) Validate() error {
-	if g.Gomod == nil {
+	if g.Gomod == nil && g.NodeMod == nil {
 		// Gomod is the only valid generator type
```

Comment:
This comment can be removed now!
```suggestion
		//
```